api/api/roles/serializers.py

```diff
@@ -11,7 +11,10 @@ class Meta:
 
 
 class ListUserRolesSerializer(serializers.ModelSerializer):
+    club__name = serializers.CharField(max_length=100)
```

Comment:
is there a reason why double underscores are used?

a better response would be
```
role: {
...
  club: {
    name: ...,
    slug: ...
  }
...
}```

Use ClubSerializer to list the club related fields.

Reply:
Instead of serializing 'foreign key club' I was extracting 'club's name & slug' that was needed so using double underscores. Current `ClubSerializer` would return other details too so I thought this was better. Will modify this anyway

Reply:
Will current approach return the format I mentioned?

Reply:
No it won't . But `ClubSerializer` will return fields other than `name & slug` too , so we need to define other custom serializer

Reply:
No doesn't strictly need to return name and slug. You can return other club fields along as well.